Resources/views/Tree/tree.html.twig

```diff
@@ -24,15 +24,23 @@ file that was distributed with this source code.
 <script type="text/javascript">
     jQuery(function($) {
         $('#tree').cmfTree({
+            dnd: { enabled: true },
             request: {
                 load: function (nodePath) {
                     return {
-                        url: '{{ path('_cmf_resource', {
+                        url: '{{ path('_get_cmf_resource', {
                             repositoryName: repository_name,
                             root_node: root_node,
```

Comment:
It's confused because we're mixing javascript and twig. It doesn't understand twig and simply assumes it's HTML (with javascript included).

These big red warnings can simply be ignored.